unittests/CppInterOp/FunctionReflectionTest.cpp

```diff
@@ -161,6 +177,15 @@ TEST(FunctionReflectionTest, HasDefaultConstructor) {
   EXPECT_TRUE(Cpp::HasDefaultConstructor(Decls[0]));
   EXPECT_TRUE(Cpp::HasDefaultConstructor(Decls[1]));
   EXPECT_FALSE(Cpp::HasDefaultConstructor(Decls[3]));
+
+  // C API
+  auto* I = clang_createInterpreterFromRawPtr(Cpp::GetInterpreter());
+  EXPECT_TRUE(clang_hasDefaultConstructor(make_scope(Decls[0], I)));
```

Comment:
Do you think we should put the tests for the c layer in a separate file?

Reply:
I don't think that's practical at the moment because of the low API coverage. We need to use a lot C++ API to set up the environment and do the test.